components/Icon/Icon.jsx

```diff
@@ -38,7 +38,7 @@ export default class Icon extends React.Component<Props> {
           title={title}
       >
         <FontAwesome
-            className={`far ${className}`}
+            className={`far fa-fw ${className}`}
```

Comment:
@thatjsguy I wanted to make you aware of this. It's kinda a hack, but it fixes the tight spacing on react icons. Let me know your thoughts.

![image](https://user-images.githubusercontent.com/863031/43326369-4e06bd94-917e-11e8-9b54-8a727e6d4575.png)

Reply:
I almost want to say that we shouldn't mix our chocolate and peanut butter (based on other places in Nitro) and use Bootstrap classes directly on `<Icon/>`. Now I don't think that `fa-fw` is a huge issue, but I have taken a different approach and decided that <span/>ning and adding those Bootstrap classes to the raw text or simply adding those classes to `<Text/>` instead is better.

Reply:
fa-fw is a font awesome class